Godeps/Godeps.json

```diff
@@ -11,7 +11,7 @@
 	"Deps": [
 		{
 			"ImportPath": "github.com/Clever/pathio",
-			"Comment": "v3.0.0-2-gc9ab5fa",
+			"Comment": "v2.0.0-23-gc9ab5fa",
```

Comment:
seems like this is just a comment. The Rev is still the same. Honestly don't really know wo this change got here.